op-node/rollup/derive/deposit_log_test.go

```diff
@@ -94,6 +94,47 @@ func makeReceipts(rng *rand.Rand, blockHash common.Hash, depositContractAddr com
 	return receipts, expectedDeposits, nil
 }
 
+func makeReceiptsSubmitCalldata(rng *rand.Rand, blockHash common.Hash, depositContractAddr common.Address, electionWinnerAddress common.Address, testReceipts []receiptData) ([]*types.Receipt, error) {
+	logIndex := uint(0)
+	receipts := []*types.Receipt{}
+	for txIndex, rData := range testReceipts {
+		var logs []*types.Log
+		status := types.ReceiptStatusSuccessful
+		if !rData.goodReceipt {
+			status = types.ReceiptStatusFailed
+		}
+		for _, isDeposit := range rData.DepositLogs {
```

Comment:
I'm not a fan of this reuse. SubmitCalldata tx is not a DepositTx, this brings in confusion. Maybe you can just add those functions in `altda_data_source_test.go`